en/sql/tuning.rst

```diff
@@ -4189,3 +4191,178 @@ The user can check the query to be cached or not by putting the session command
     }
 
 The cached query is shown as **query_string** in the middle of the result screen. Each of the **n_entries** and **n_pages** represents the number of cached queries and the number of pages in the cached results. The **n_entries** is limited to the value of configuration parameter **max_query_cache_entries** and the **n_pages** is limited to the value of **query_cache_size_in_pages**. If the **n_entries** is overflown or the **n_pages** is overflown, some victims among the cache entries are selected and they are uncached. The number of victims is about 20% of **max_query_cache_entries** value and of the **query_cache_size_in_pages** value.
+
+.. _correlated-subquery-cache:
+
+SUBQUERY CACHE (correlated)
+------------------------------------
+
+Subquery cache optimization can be used to enhance the performance of queries containing correlated subqueries, and the results of the subqueries are cached in independent spaces for each subquery.
+To disable subquery cache optimization, use the NO_SUBQUERY_CACHE hint on the target subquery.
+
+Subquery cache does not operate in the following scenarios:
+
+* When the correlated subquery contains another correlated subquery.
+* When the subquery is not in the SELECT clause.
+* When the subquery includes OID-related features.
+* When the subquery includes the NO_SUBQUERY_CACHE hint.
+* When storing new results exceeds the set subquery cache size (default: 2MB).
+* When the subquery contains functions that change results with each execution, such as random() or sys_guid().
+
+If the correlated subquery is in the SELECT clause, subquery cache is utilized.
+Among the recurrently executed correlated subqueries, if the column values referenced in the main query remain the same, the cached results are used to prevent re-execution.
```

Comment:
```suggestion
Among the recurrently executed correlated subqueries, if the column values referenced in the main query remain the same with previously cached value, the cached results are used to prevent re-execution.
```